api/persistence.go

```diff
@@ -0,0 +1,22 @@
+package api
+
+import (
+	"encoding/json"
+	"syscall"
+)
+
+
+type Persistence interface {
```

Comment:
I don't really understand why this is called `Persistence`.

Reply:
In cc-runtime, the same functionality is called "filesystem". And even in runv-cli, the data is also saved on the local filesystem. But the name of the media for saving the data doesn't represent the functionality well. In other cases, we might save the data in the DB(hyperd,fratki) or k8s-etcd.

The functionality is "retrieving/dumping the data from running sandbox + saving it on some places + restructuring the running sandbox (golang objects) for later operations". `Persistence` is competent at describing it. I don't insist on this name if some other name is proposed. And I can change it to "Filesystem" if it gets some votes.